File: Slip 3/Core Java/program2.java
class program2 {
    public static void main(String args[]) {
        Cone obj = new Cone(5, 10);
        System.out.println("Area of cone: " + obj.area());
        System.out.println("Volume of cone: " + obj.volume());

        Cylinder obj1 = new Cylinder(5, 10);
        System.out.println("Area of cylinder: " + obj1.area());
        System.out.println("Volume of cylinder: " + obj1.volume());
    }
}

abstract class Shape {
    abstract float area();

    abstract float volume();
}

class Cone extends Shape {
    float radius, height;

    Cone(float r, float h) {
        super();
        radius = r;
        height = h;
    }

    float area() {
        return (float) (3.14 * radius * (radius + (radius * radius) + (height * height)));
    }

    float volume() {
        return (float) (3.14 * radius * radius * height / 3);
    }
}

class Cylinder extends Shape {
    float radius, height;

    Cylinder(float r, float h) {
        super();
        radius = r;
        height = h;
    }

    float area() {
        return (float) (2 * 3.14 * radius * (radius + height));
    }

    float volume() {
        return (float) (3.14 * radius * radius * height);
    }
}

File: Slip 19/Python/program2.py
class Shape:
    def area(self):
        return 0
    def volume(self):
        return 0

class Square(Shape):
    def __init__(self, length):
        self.length = length
    def area(self):
        return self.length ** 2
    def volume(self):
        return self.length * self.length * self.length

class Circle(Shape):
    def __init__(self, radius):
        self.radius = radius
    def area(self):
        return 3.14 * self.radius ** 2
    def volume(self):
        return 0

obj = Square(5)
print("Area of square:", obj.area())
print("Volume of square:", obj.volume())

obj = Circle(3)
print("Area of circle:", obj.area())
print("Volume of circle:", obj.volume())
